instagram-app/src/App.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import dummyData from "./data/dummy-data"
+import "./App.scss"
+
+import PostContainer from './components/PostContainer/PostContainer';
+import SearchBar from './components/SearchBar/SearchBar'
+
+import { Container } from 'reactstrap';
+
+class App extends Component {
```

Comment:
you don't actually need the constructor or the super here. you could simply write this as

```
class App extends Component {
  state = {
      data: dummyData
  };
  console.log(this.state.data);
```